avi.wad.Organizer.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id" : "avi.wad.Organizer",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "organizer",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home",
```

Comment:
You already have access to all xdg-* folders if you have access to home directory.

Reply:
So I should remove all xdg references in the manifest? Also wow that was a really fast response!

Reply:
done

Reply:
What do you need access to the home directory for? Could the app work with the portal?

Reply:
I need the home directory because my app takes a folder of the user's choice and attempts to organize the files within it. From what I know, portals only allow the user to choose a file, not folders.